data/posts/rr-governance.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "React Router Open Governance"
+summary: "React Router is moving to an Open Governance Model"
+date: 2025-06-04
+image: /blog-images/headers/react-router-v7.jpg
+imageAlt: React Router Logo
+authors:
+  - Matt Brophy
+  - Brooks Lybrand
+---
+
+Michael Jackson, co-creator of React Router, [said it best](https://x.com/mjackson/status/1927739177149382991):
+
+> React Router isn't just mine and Ryan's baby anymore. It is a mature OSS project with millions of dependents. We want everyone to have a say in how the project moves forward.
+
+React Router has been around for over 10 years under the development and oversight of [Michael](https://x.com/mjackson) and [Ryan](https://x.com/ryanflorence). When they obtained funding for Remix in 2021 and built a team around Remix, they also indirectly built a team around React Router. That team has been working on React Router for a few years now and it's time to formalize the process we've loosely been using internally for a while now.
+
+Last week in our ["Wake Up, Remix!"](https://remix.run/blog/wake-up-remix) post, we announced a new [Open Governance Model](https://github.com/remix-run/react-router/blob/main/GOVERNANCE.md) for React Router. We planned to announce these in the reverse order: first the governance model so folks knew that React Router wasn't going anywhere, followed by our plans for Remix v3.
+
+However, a leak of an old version of the Remix announcement (you can go find that yourself) forced our hand and we had to get that post out quicker than anticipated. This meant we didn't get to give the new governance model the proper focus and announcement, so we're doing that now. We think it's just as exciting for the community and the future of React Router!
+
+## Motivation
+
+React Router has gone through many major evolutions in its 10+ year lifetime, many of those dictated by the evolution of React itself (i.e., the introduction of hooks). Recently, with the creation of the Remix framework and its integration into React Router "Framework Mode" in v7, the API surface has increased even more. With the introduction of React 19 and [(soon) RSC support](./rsc-preview), aspects previously handled by React Router are going to be able to be handled by React.
```

Comment:
```suggestion
React Router has gone through many major evolutions in its 10+ year lifetime, many of those dictated by the evolution of React itself (i.e., the introduction of hooks). Recently, with the creation of the Remix framework and its integration into React Router "Framework Mode" in v7, the API surface has increased even more. And with the arrival of React 19 and [(soon) RSC support](./rsc-preview), React taken on aspects previously handled by React Router.
```

Reply:
Thanks! Grammar was a bit off so I adjusted and pushed up a change